src/Controller/CampaignMonitorController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\campaignmonitor\Controller\CampaignMonitorController.
+ */
+
+namespace Drupal\campaignmonitor\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Form\FormBuilder;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Url;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Drupal\campaignmonitor\CampaignMonitor;
+
+class CampaignMonitorController extends ControllerBase {
+
+  /**
+   * The campaign monitor.
+   *
+   * @var \Drupal\campaignmonitor\CampaignMonitor
+   */
+  protected $campaignMonitor;
+
+  /**
+   * Settings for the module.
+   */
+  protected $settings;
+
+  /**
+   * Constructs a new SubscribeForm.
+   */
+  public function __construct() {
+    $this->campaignMonitor = CampaignMonitor::GetConnector();
+    $this->settings = \Drupal::config('campaignmonitor.general')->get();
```

Comment:
If you'd prefer to do this manually, you can see an example at https://www.drupal.org/node/2203931